templates/Layout/Page_results.ss

```diff
@@ -0,0 +1,79 @@
+<div class="content search-results">
```

Comment:
@MasseyIsaako why do we need this file here? I checked the starter theme and this matches that template

Reply:
Yeah, I did duplicate that template from Starter. I needed to apply a minor styling change. Also, just prepping it for the [new design](https://projects.invisionapp.com/d/main#/console/15471951/331473579/preview) too, as it looks pretty different from what is currently in the Starter theme. I thought it would be better to separate them from each other. Is there is a better way of handling this?